### DISCUSSION TITLE
is BentoML Grafana dashboard json available?

### DISCUSSION THREAD

**Discussion by Amitg1** [bentoml/BentoML#4305] [General]
Hi, 
We are looking for a Grafana dashboard(json) based on metrics exposed by the BentoML app.
I'm aware I can build it myself, looking to save time.

Thanks.

**Comment by Haivilo** [comment]
Here is the Grafana dashboard json for BentoML app used in Yatai.
https://github.com/bentoml/Yatai/blob/main/scripts/monitoring/bentodeployment-dashboard.json

This documentation describes how it is used in Yatai:
https://docs.yatai.io/en/latest/observability/metrics.html
